schemes/rrtmgp/rrtmgp_inputs.F90

```diff
@@ -64,7 +59,7 @@ subroutine rrtmgp_inputs_run(dosw, dolw, snow_associated, graupel_associated, &
      real(kind_phys),                      intent(in) :: stebol                ! Stefan-Boltzmann constant (W m-2 K-4)
      type(ty_gas_optics_rrtmgp_ccpp),      intent(in) :: kdist_sw              ! Shortwave gas optics object
      type(ty_gas_optics_rrtmgp_ccpp),      intent(in) :: kdist_lw              ! Longwave gas optics object
-     character(len=*), dimension(:),       intent(in) :: gaslist_lc            ! Radiatively active gases
+     character(len=5), dimension(:),       intent(in) :: gaslist_lc            ! Radiatively active gases
```

Comment:
Is there a reason the `*` was replaced with `5` here?  If not then it might be good to switch back, as `*` would provide more flexibility in case we ever had to change it.